src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/helpers.js

```diff
@@ -11,6 +11,15 @@ function assignPaymentMethodValue() {
   ).innerHTML;
 }
 
+function assignOrderFormValues(response) {
```

Comment:
maybe we can find a better name for the function.. something like saveOrderIdentifiers?